### DISCUSSION TITLE
Trashed filter does not show on filter count

### DISCUSSION THREAD

**Discussion by dmason30** [laravel/nova-issues#6209] [Ideas & Feature Requests]
- Nova Version: 4.32.11

### Description:
Selecting any option in the trashed filter does not get included in the applied filter count displayed next to the filter dropdown button.

<img width="280" alt="image" src="https://github.com/laravel/nova-issues/assets/20278756/6f2ede9c-d28e-4cdc-a074-ec6928a821e4">

Expected:
<img width="95" alt="image" src="https://github.com/laravel/nova-issues/assets/20278756/24c0e845-267b-408f-b4cf-fc760900d586">

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
